hw6/tasks/task2.py

```diff
@@ -0,0 +1,36 @@
+"""
+Задача 2: Котики в помощь 🐈 +1
+
+В интернете часто шутят, что котики делают всё лучше.
+Например, говорят: 'Хочешь, чтобы презентация прошла успешно? Добавь котиков!'
+
+У одного преподавателя есть кошка по имени Маруся.
+Напишите программу, которая будет реагировать на упоминание её имени.
+
+Создайте функцию 'get_answer', которая принимает на вход строку.
+Если в строке встречается слово 'Маруся' в любом регистре,
+например, 'маруся', 'МаРуСя' и т.д., то функция должна вернуть строку 'Мяу...'.
+Если слова "Маруся" в строке нет, то функция должна вернуть строку '...'.
+
+Под блоком if __name__ == '__main__':
+1. Получите от пользователя строку.
+2. Выполните функцию, передав ей нужные аргументы.
+3. Распечатайте результат вызова функции.
+"""
+
+
+def get_answer(question):
+    question_lower = question.lower()
+
+    if "маруся" in question_lower:
+        return "Мяу..."
+    else:
+        return "..."
```

Comment:
Хорошее решение, можно оптимизировать:
```suggestion
    if "маруся" in question.lower():
        return "Мяу..."
    return "..."
```